client/api_key.go

```diff
@@ -5,11 +5,46 @@ package client
 
 import (
 	"fmt"
+	"github.com/go-openapi/strfmt"
+	log "github.com/sirupsen/logrus"
+	"time"
 
 	clientv1 "github.com/spectrocloud/palette-sdk-go/api/client/v1"
 	"github.com/spectrocloud/palette-sdk-go/api/models"
 )
 
+// CreateAPIKey creates a new API key for the specified tenant.
+func (h *V1Client) CreateAPIKey(name, TenantUID, emailID, org, password string) (string, error) {
+	annotations := map[string]string{
```

Comment:
This is a public repo. Please make annotations a param and remove the hardcoded QA annotations.

Reply:
> This is a public repo. Please make annotations a param and remove the hardcoded QA annotations.

Removed the hardcoded annotations